packages/jellyfish-api-core/__tests__/category/masternode/getMasternode.test.ts

```diff
@@ -0,0 +1,63 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { MasternodeState } from '../../../src/category/masternode'
+
+describe('Masternode', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  it('should getMasternode', async () => {
+    let id: string = ''
+
+    const ownerAddress = await client.wallet.getNewAddress()
+    await client.masternode.createMasternode(ownerAddress)
+
+    await container.generate(1)
+
+    const masterNodes = await client.masternode.listMasternodes()
+
+    for (const mnId in masterNodes) {
+      if (masterNodes[mnId].ownerAuthAddress === ownerAddress) {
+        id = mnId
+      }
+    }
+
+    const masternode = await client.masternode.getMasternode(id)
```

Comment:
```suggestion
    const id = await client.masternode.createMasternode(ownerAddress)
    await container.generate(1)

    const masternode = await client.masternode.getMasternode(id)
```

createMasternode returns a txid, which is also the masternode id

Reply:
Did not know about that. Let me refactor